### PR TITLE
Transition update!

### PR DESCRIPTION
**Pull request by mrose17**
1. Delete old owners at the end of the process (previous code tried to do this _in situ_ and missed quite a few).
2. Be sure to update the `authority` field for owners and publishers to the new ownerId.

### CODE REVIEW

File: eyeshade/controllers/owners.js [hunk 1]
```diff
@@ -187,8 +188,10 @@ v3.bulk = {
           entry = await publishers.findOne({ publisher: channelId })
           if (!entry) continue
 
-          await publishers.update({ publisher: channelId }, { $set: { owner: owner.owner_identifier } },
-                                  { upsert: true })
+          if ((entry.owner) && (cleanup.indexOf(entry.owner))) cleanup.push(entry.owner)
```
Comment (ayumi) [inline eyeshade/controllers/owners.js]: `cleanup` starts as `[]` so isn't `cleanup.indexOf(entry.owner)` going to return false and prevent pushing of owners to cleanup every time?

File: eyeshade/controllers/owners.js [hunk 2]
```diff
@@ -187,8 +188,10 @@ v3.bulk = {
...
+          await publishers.update({ publisher: channelId }, {
+            $set: { owner: owner.owner_identifier, authority: owner.owner_identifier }
```
Comment (ayumi) [inline eyeshade/controllers/owners.js]: why do we set two properties both to the same value?

Comment (mrose17) [reply eyeshade/controllers/owners.js]: in the future, there may be a different entity that authorizes an owner...

File: eyeshade/controllers/owners.js (hunk 1 above)
Comment (ayumi) [reply eyeshade/controllers/owners.js]: * updated my comment above

Comment (mrose17) [reply eyeshade/controllers/owners.js]: that's a typo. fixed

**Pull request merged**